sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/AADClientRegistrationRepository.java

```diff
@@ -68,6 +71,14 @@ public static boolean isDefaultClient(ClientRegistration clientRegistration) {
             clientRegistration.getClientName());
     }
 
+    public boolean isClientCredentials(String clientId) {
```

Comment:
clientId? I think it should be registrationId if it call `allClients.get()`

Reply:
I have fixed it just now